### ISSUE TITLE
Show highlighted messages in a notification popup on the bottom right

### ISSUE THREAD

**Issue by fourtf** [Chatterino/chatterino2#398]

**Comment by fourtf** [comment]
started in highlightcontroller

**Comment by zneix** [comment]
I assume that by "bottom right notifications" you've meant native Windows notifications handled by the system. On Linux, there are notification managers and my question is: Would supporting those and sending notifications via `notify-send` be in scope of Chatterino project?

**Comment by leon-richardt** [comment]
> Would supporting those and sending notifications via `notify-send` be in scope of Chatterino project?

That's pretty much what #1985 was all about

**Comment by zneix** [comment]
> > Would supporting those and sending notifications via `notify-send` be in scope of Chatterino project?
> 
> That's pretty much what #1985 is all about

Mentioned PR is creating custom notifications built from scratch, however I'd rather prefer having notification system integrated and unified with my dunst notifications - just like most other programs I use (e.g. thunderbird, lutris, discord, bleachbit, konsole, custom scripts, etc.).

**Comment by leon-richardt** [comment]
Yes, you are correct! My apologies, for some reason I thought the implementation made use of `notify-send` but maybe that was part of the previous iteration of the PR.

Nonetheless, I think we should decide on a single way to handle notifications (either going with `notify-send` or with @apa420's custom implementation).

**Comment by apa420** [comment]
The previous iteration of the rework did use notify-send, the only issue was I was unable to replicate how to compile the program on windows
